### ISSUE TITLE
Float division by zero (kappa)

### ISSUE THREAD

**Issue by Chrisini** [zhuanjiao2222/DRAC2022#1]
Quadratic weighted kappa: Zero division error should be fixed. Same numbers produce nan in the scikit implementation (https://scikit-learn.org/stable/modules/generated/sklearn.metrics.cohen_kappa_score.html), but throw an error here.

Example inputs: 
gt tensor([0, 0, 0, 0]) # rater 1 aka ground truth label
max_output [0. 0. 0. 0.] # rater 2 aka model output class

Error message:
/shared-libs/python3.9/py/lib/python3.9/site-packages/sklearn/metrics/_classification.py:663: RuntimeWarning: invalid value encountered in double_scalars
  k = np.sum(w_mat * confusion) / np.sum(w_mat * expected)
/shared-libs/python3.9/py/lib/python3.9/site-packages/numpy/lib/function_base.py:518: RuntimeWarning: Mean of empty slice.
  avg = a.mean(axis, **keepdims_kw)
/shared-libs/python3.9/py/lib/python3.9/site-packages/numpy/core/_methods.py:190: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
division by zero
